db/data-generator.js

```diff
@@ -0,0 +1,89 @@
+const faker = require('faker');
+const fs = require('fs');
+const moment = require('moment');
+
+const generateRandomNumber = function(min, max, decimal) {
+  decimal = decimal || 0;
+  let rand = Math.random() * (max - min) + min;
+  let power = Math.pow(10, decimal);
+  return Math.floor(rand * power) / power;
+};
+
+const generateRandomDate = function(start, end) {
```

Comment:
Same here, might be able to use 'faker.date.future' or some of the other faker date methods

Reply:
`Faker.date.future()` and `faker.date.soon()` didn't work out for me yesterday, hence the functions!